src/base/ApplicationECSAlbCodeDeploy.ts

```diff
@@ -11,6 +11,9 @@ export interface ApplicationECSAlbCodeDeployProps {
   targetGroupNames: string[];
   tags?: { [key: string]: string };
   dependsOn?: TerraformResource[];
+  notifyOnStarted?: boolean;
+  notifyOnSucceeded?: boolean;
+  notifyOnFailed?: boolean;
```

Comment:
Suggestion to consolidate as below:
```suggestion
  notifications: {
    onStarted?: boolean;
    onSucceeded?: boolean;
    onFailed?: boolean;
  }
```